shell/probe/raindrop.ps1

```diff
@@ -52,11 +52,11 @@ while ($true) {
             exit 1
         } else {
             $Dat = ""
-            Remove-Item $Dir -Force -Recurse
             Start-Sleep -Seconds $Sleep
         }
     } catch { 
         Write-Error $_
         Start-Sleep -Seconds $Sleep
     }
+    Remove-Item $Dir -Force -Recurse -ErrorAction SilentlyContinue
```

Comment:
this will delete the directory after each test vs every test loop/iteration. thoughts on this change?

Reply:
is it still a requirement that raindrop mimic nocturnal exactly?

Reply:
yes

Reply:
> this will delete the directory after each test vs every test loop/iteration. thoughts on this change?

I don't like that it's done after the sleep


> is it still a requirement that raindrop mimic nocturnal exactly?

It should

Reply:
It's the root cause of this https://detectors.atlassian.net/browse/DET-299?atlOrigin=eyJpIjoiYjlkZjM2ZWI0YTk2NGEwZWIwYjg2M2M5MGJmYjM0NTAiLCJwIjoiamlyYS1zbGFjay1pbnQifQ

Reply:
I can move it before the sleeps, but the deletion MUST happen no matter what. We are currently not ensuring that and leaving artifacts on the system

Reply:
Pushed a commit to delete before sleep

Reply:
> is it still a requirement that raindrop mimic nocturnal exactly?

We would need to bring Nocturnal in line if this change is accepted @makk94

Reply:
nocturnal looks good as is